### ISSUE TITLE
Navbar bug

### ISSUE THREAD

**Issue by piechade** [mopsalarm/Pr0#217]
Kann man was gegen diesen leicht nervigen Bug machen? Die App rutscht immer unter die Navbar.

![Screenshot_20200322-125743](https://user-images.githubusercontent.com/1434005/77248829-e4c25500-6c3c-11ea-86d9-ae332629476f.jpg)

**Comment by mopsalarm** [comment]
Ja, das ist ein Problem. Habe ich auf dem Plan, aber bisher noch keine Zeit gehabt, das zu fixen.
